SimpleFTP/SimpleFTP.Tests/FTPTests.cs

```diff
@@ -0,0 +1,124 @@
+// <copyright file="FTPTests.cs" company="Gorlov Kirill">
+// Copyright (c) Gorlov Kirill. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the repository root for license information.
+// https://github.com/GolrovKirill/SPbU-Prog3/blob/main/LICENSE
+// </copyright>
+namespace SimpleFTP.Tests;
+
+using System.Net;
+
+/// <summary>
+/// Unit tests for the FTP server functionality.
+/// </summary>
+public class FTPTests
+{
+    private const int Port = 1234;
+
+    private const string Host = "localhost";
+
+    private FTPServer server;

```

Comment:
```suggestion
    private FTPServer? server;

```
нужно сделать вот так, чтобы StyleCop не ругался